packages/mosaic/tree/tree-option.component.ts

```diff
@@ -18,6 +18,11 @@ import { CdkTreeNode } from '@ptsecurity/cdk/tree';
 import { CanDisable, toBoolean } from '@ptsecurity/mosaic/core';
 
 
+export enum MultipleMode {
+    CHECKBOX = 'checkbox',
+    KEYBOARD = 'keyboard'
+}
+
```

Comment:
this code used only in tree-selection why do you place it there ?

Reply:
its used in packages/mosaic/tree/tree-option.component.ts as well

![image](https://user-images.githubusercontent.com/3306378/67552332-8a67ab80-f713-11e9-8df6-e5e304e3407b.png)

btw there is reference to `tree-option.component.ts` at `tree-selection.component.ts` and if i import `tree-selection.component.ts` in `tree-option.component.ts` i get circular dependency.
If you want to have it out of `tree-option.component.ts` I could offer move it into separated file.

Beside it i believe mode mostly is related to option rather than selection at all